PlantDiary21FS7024002/Pages/Index.cshtml.cs

```diff
@@ -77,11 +72,13 @@ public void OnGet()
                 // This was generated by QuickType.
                 SpecimenCollection specimenCollection = specimenfeed.SpecimenCollection.FromJson(specimenJSON);
                 // get the raw list of specimesn from the specimen collection.
-                List < Specimen > specimens = specimenCollection.Specimens;
-                JSchema schema = JSchema.Parse(System.IO.File.ReadAllText("SpecimenSchema.json"));
+                List<Specimen> specimens = specimenCollection.Specimens;
+                string rawSchema = System.IO.File.ReadAllText("SpecimenSchema.json");
+                JSchema schema = JSchema.Parse(rawSchema);
```

Comment:
It's risky to nest multiple method calls on one line, so I split each to its own line.
Source: https://softwareengineering.stackexchange.com/questions/133404/what-is-the-ideal-length-of-a-method-for-you